packages/perseus/src/components/__stories__/graph.stories.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from "react";
 
 import Graph from "../graph";
-// TODO(scottgrant): Katex is unavailable here. Fix!
```

Comment:
I couldn't even make sense of what was to be done here so I'm deleting this comment.